### DISCUSSION TITLE
Missing arguments in stacktrace

### DISCUSSION THREAD

**Discussion by leoleva** [getsentry/sentry-php#1791] [Q&A] [answered]
Hey, sometimes when sentry handles errors in stack trace in some apps we see arguments, in others we don't (php slim vs Symfony integration). I'm wondering how to enable it so arguments would appear every time? Thanks

**Comment by cleptric** [comment, marked as answer]
Make sure `zend.exception_ignore_args` is set to `Off` in your php.ini.
https://docs.sentry.io/platforms/php/troubleshooting/#missing-variables-in-stack-traces
